README.md

```diff
@@ -16,30 +23,57 @@ The application uses Ruby 2.5.0. You should install it to run the application. (
 We use PostgreSQL as database. You should install it to run the app.
 If needed, you should configure the `config/database.yml` file.
 
-### Configuring the project
+> Tip
+>
+> You can use docker to easy database setup
+> ```bash
+> $ docker run --name postgres_open_jobs \
+>   -e POSTGRES_PASSWORD=postgres \
+>   -e POSTGRES_DB=opensanca_jobs_development \
+>   -e POSTGRES_USER=postgres \
+>   -p 5432:5432 -d postgres:9.6.5
+> ```
 
-Clone it
+#### Redis
```

Comment:
There is a `Redis` level but there is not a Postgres level. 🤔

I like your idea. So, I think we should add all with levels like:
```
#### Postgres
... (and here you can move the comment of postgres in dependencies)
#### Redis
#### Mail server
For local development, we suggest to use Mailhog (https://github.com/mailhog/MailHog) or Mailcatcher (https://mailcatcher.me/)
```

Reply:
good